src/components/Editable.js

```diff
@@ -348,6 +378,8 @@ const Editable = ({ disabled, isEditing, thoughtsRanked, contextChain, cursorOff
    * Prevented by mousedown event above for hidden thoughts.
    */
   const onFocus = e => {
+    setTimeout(makeToolbarPositionFixed, 100)
```

Comment:
What is the purpose of the delay, and does the length of time matter?

Reply:
Author of this solution said that delay is required, but it works for 6s even without delay.  I don't think that length matters. I removed timeout. Does it still work on your device?

Reply:
Removing the `setTimeout` works on iPhone 11, but only when applied to f4e3dfe54dc31f2368584c78966a8d8b0016a5d2. Something broke in 326233aa.